pytest.ini

```diff
@@ -0,0 +1,39 @@
+[pytest]
+markers =
+    pep8: workaround for https://bitbucket.org/pytest-dev/pytest-pep8/issues/23/
+
+python_files =
+    test_*.py
+
+addopts =
+    --cov ofxstatement.configuration
+		--cov ofxstatement.exceptions
+		--cov ofxstatement.ofx
+		--cov ofxstatement.parser
+    --cov ofxstatement.plugin
+    --cov ofxstatement.statement
+    --cov ofxstatement.tool
+    --cov ofxstatement.ui
```

Comment:
Listing out all the modules is not scalable and pretty brittle. 

BTW, there's `make coverage` target.